routes/api.php

```diff
@@ -1,11 +1,16 @@
 <?php
 
 use App\Http\Controllers\CompanyController;
+use App\Http\Controllers\LoginController;
+use App\Http\Controllers\RegisterController;
 use App\Http\Controllers\RequestedSongController;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Route;
 
 
+
+
+
```

Comment:
Humans need extra space but the code don't need more than 1 line 🙃

Reply:
done!